src/main/java/de/rwth/idsg/steve/config/BeanConfiguration.java

```diff
@@ -266,6 +267,7 @@ public void extendMessageConverters(List<HttpMessageConverter<?>> converters) {
      * {@link WebMvcConfigurationSupport#requestMappingHandlerAdapter(ContentNegotiationManager, FormattingConversionService, org.springframework.validation.Validator)}.
      */
     @Bean
+    @Qualifier("steveObjectMapper")
```

Comment:
As you only use it in security config, maybe you should move it security config too.

Reply:
i prefer to have this objectmapper in close proximity to `extendMessageConverters(..) ` where i modify and configure it.